processor/src/main/java/org/mapstruct/ap/internal/model/source/ForgedMethod.java

```diff
@@ -82,11 +81,11 @@ public ForgedMethod(String name, Type sourceType, Type returnType, MapperConfigu
      * @param mappingOptions the mapping options for this method
      * @param forgedNameBased forges a name based (matched) mapping method
      */
-    public ForgedMethod(String name, Type sourceType, Type returnType, MapperConfiguration mapperConfiguration,
-                        ExecutableElement positionHintElement, List<Parameter> additionalParameters,
-                        ParameterProvidedMethods parameterProvidedMethods, ForgedMethodHistory history,
-                        MappingOptions mappingOptions, boolean forgedNameBased) {
-        String sourceParamName = Strings.decapitalize( sourceType.getName() );
+     public ForgedMethod(String name, Type sourceType, Type returnType, MapperConfiguration mapperConfiguration,
```

Comment:
Are this formatting changes really needed?

Reply:
No they aren't.. I'll remove them..

Reply:
I did a lot of trial and error here.. So I guess there was (in one of my commits) an extra param..

Reply:
makes sense